sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -395,5 +587,163 @@ public async Task StartRecognizeCustomFormsFromUriThrowsForNonExistingContent(bo
             Assert.True(operation.HasValue);
             Assert.AreEqual(0, operation.Value.Count);
         }
+
+        private void ValidateFormPage(FormPage formPage, bool includeTextContent, int expectedPageNumber)
+        {
+            Assert.AreEqual(expectedPageNumber, formPage.PageNumber);
+
+            Assert.AreEqual(LengthUnit.Inch, formPage.Unit);
```

Comment:
If we pass a jpg instead of a pdf, the unit will be Pixels.

Reply:
You are right. This code was originally part of one of the tests, so it made sense at that point. In this case, I think it's reasonable to skip the verification, since this method is more general, and units are already being tested in other methods.